### DISCUSSION TITLE
Issue when calling an automation?

### DISCUSSION THREAD

**Discussion by 4irm** [Blackymas/NSPanel_HA_Blueprint#2335] [General]
I am running 4.3.8, I linked an automation on the button page - actually starting a cleaning task of my vaccum calling a service/action.

When hitting the button on the panel usually nothing happens, however, when triggering the automation via home assistant UI directly from the automation menu the automation works flawlessly.

Any clue? Thank you!

**Comment by sanvalerius** [comment]
That's probably because by default it "switch" on/off the automation instead of "trigger" the automation.
You have to change the action to execute the automation instead.

**Reply by 4irm** [reply]
Thanks for the feedback, maybe I have just found the solution...

<img width="808" alt="image" src="https://github.com/user-attachments/assets/f400ea2a-634c-4a18-9350-400519045df9">
